src/test/java/gov/healthit/chpl/aqa/pageObjects/UserManagementPage.java

```diff
@@ -0,0 +1,21 @@
+package gov.healthit.chpl.aqa.pageObjects;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+/**
+ * Class UserManagementPage definition.
+ */
+public final class UserManagementPage extends BasePage {
+    private UserManagementPage() {
+    }
+
+    public static WebElement impersonateRole(final WebDriver driver, final String impersonateRoleName) {
+        return driver.findElement(By.xpath("//span[contains(text(),'" + impersonateRoleName + "')]"));
+    }
+
+    public static WebElement impersonateRoleUsername(final WebDriver driver) {
+        return driver.findElement(By.xpath("//button[@id='login-toggle']"));
```

Comment:
I think this element already exists, but even if it doesn't, it wouldn't be specific to the user management page, so should be in the base page definition

Reply:
this element does exist in LoginPage